README.md

```diff
@@ -16,20 +16,59 @@ Note: The steps below will have to be run when either of the following happen:
 
 ### iOS Simulator
 
-1. `yarn build:ios`
+1. `yarn build:local:ios`
 2. double click newly created `*.tar.gz` file in `build\` directory of project
 3. you'll now see a `ChainReactApp2023.app` file in the root of your project > drag and drop this file to your simulator
 4. `yarn start`
 5. `i` > app should now open and build
 
 ### Android Emulator
 
-1. `yarn build:android`
+1. `yarn build:local:android`
 2. APK will be generated in the `build\` directory of the project
 3. Drag and drop this file to your emulator or sideload to Android device of choice
 4. `yarn start`
 5. `a` > app will open on the emulator or connected device (use `adb devices` to see if properly connected)
 
+### Running on devices
+
+In order to run on an iOS device, you'll need to add your device to EAS.
+
+1. Run `eas device:create` and follow the prompts.
+2. Enter your UDID or use the web URL option
+3. Make sure to give the device a descriptive name (e.g. Jamon's iPhone 12)
+
+#### Running on device in development mode
+
+Great for developing on device for testing features like screenreader support
+
+Android
+
+1. Follow emulator directions above
+2. Instead of dragging build to emulator, use `adb install` to install on connected device
+
+iOS
+
+1. Run `yarn build:dev:device`
```

Comment:
I would add in that you can optionally pass `--local` to not wait on the EAS servers if you want to build locally

Reply:
Does that work for physical iOS devices? I've never installed a plain .ipa file to a device without XCode

Reply:
Updated to include a local option!